### PR TITLE
release_22.01 configuration updates

### PR DESCRIPTION
**Pull request by gmauro**
This PR contains playbook updates for the release_22.01

### CODE REVIEW

File: files/galaxy/config/datatypes_conf.xml [hunk 1]
```diff
@@ -29,8 +29,8 @@
     <datatype extension="arb" auto_compressed_types="gz" type="galaxy.datatypes.binary:Binary" subclass="true" display_in_upload="true"/>
     <datatype extension="arff" type="galaxy.datatypes.text:Arff" mimetype="text/plain" display_in_upload="true"/>
     <datatype extension="paf" auto_compressed_types="gz" type="galaxy.datatypes.text:Paf" mimetype="text/plain" display_in_upload="true"/>
-    <datatype extension="gfa1" auto_compressed_types="gz" type="galaxy.datatypes.text:Gfa1" mimetype="text/plain" display_in_upload="true"/>
```
Comment (bgruening) [inline files/galaxy/config/datatypes_conf.xml:32]: @gallardoalba this was needed for VGP, wasn't it? Do we have upstream it?

File: group_vars/gxconfig.yml [hunk 2]
```diff
@@ -286,6 +283,15 @@ base_app_main: &BASE_APP_MAIN
...
   tool_sheds_config_file: "{{ galaxy_config_dir }}/tool_sheds_conf.xml"
 
+  # This option controls whether legacy datatypes are loaded from
+  # installed tool shed repositories. We're are in the process of
+  # disabling Tool Shed datatypes. This option with a default of true
+  # will be added in 22.01, we will disable the datatypes on the big
+  # public servers during that release. This option will be switched to
+  # False by default in 22.05 and this broken functionality will be
+  # removed all together during some future release.
+  #load_tool_shed_datatypes: true
```
Comment (bgruening) [inline group_vars/gxconfig.yml:293]: I guess we can switch this to false later this week and see how many people cry - I expect none.

File: group_vars/gxconfig.yml [hunk 3]
```diff
@@ -497,7 +503,7 @@ base_app_main: &BASE_APP_MAIN
   # to this directory.
   # The value of this option will be resolved with respect to
   # <data_dir>.
-  cluster_files_directory: /data/dnb01/galaxy_db/pbs
+  #cluster_files_directory: pbs
```
Comment (bgruening) [inline group_vars/gxconfig.yml:506]: Is that on purpose?

Comment (gmauro) [reply group_vars/gxconfig.yml:506]: yes, should be not needed anymore

File: files/galaxy/config/datatypes_conf.xml (hunk 1 above)
Comment (gallardoalba) [reply files/galaxy/config/datatypes_conf.xml:32]: Yes, it is also upstream.

Comment (bgruening) [reply files/galaxy/config/datatypes_conf.xml:32]: but for the 21.01 release we will need it here, I guess?

Comment (gmauro) [reply files/galaxy/config/datatypes_conf.xml:32]: I merged it from upstream
https://github.com/galaxyproject/galaxy/blob/603122bf2a3c8308dfc1f3d2686ba9f2b2841a68/lib/galaxy/config/sample/datatypes_conf.xml.sample#L31

Comment (bgruening) [reply files/galaxy/config/datatypes_conf.xml:32]: @gallardoalba can you please check where this was lost? Or maybe never upstreamed.

Comment (gallardoalba) [reply files/galaxy/config/datatypes_conf.xml:32]: It is in the dev branch of the Galaxy repo https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/datatypes_conf.xml.sample#L31; I thought that it means to be upstream; I guess that this is not what you were referring?

Comment (gallardoalba) [reply files/galaxy/config/datatypes_conf.xml:32]: Ah, it doesn't include the compression.

Comment (gallardoalba) [reply files/galaxy/config/datatypes_conf.xml:32]: Yes, you are right, it wasn't merged finally https://github.com/galaxyproject/galaxy/pull/13501. I cannot understand why it was closed.

Comment (bgruening) [reply files/galaxy/config/datatypes_conf.xml:32]: That was a mistake I think. Please reopen that PR.

Comment (mvdbeek) [reply files/galaxy/config/datatypes_conf.xml:32]: I have not closed this. In fact it doesn't say who or why the PR was closed, quite mysterious.

Comment (gallardoalba) [reply files/galaxy/config/datatypes_conf.xml:32]: > I have not closed this. In fact it doesn't say who or why the PR was closed, quite mysterious.

Thanks Marius! I'll open it again.

Comment (mvdbeek) [reply files/galaxy/config/datatypes_conf.xml:32]: Actually I can tell you why, the commit landed in dev via another PR, so there was nothing to merge as the PR was still targeting dev.

**Pull request merged**